app/pages/resource/resource-info/ResourceInfo.js

```diff
@@ -18,9 +18,14 @@ function ResourceInfo({
 
   return (
     <section className="app-ResourceInfo">
-      <div className="app-ResourceInfo__description">
-        {resource.description && <WrappedText openLinksInNewTab text={resource.description} />}
-      </div>
+      {resource.description && (
+        <ResourcePanel header={t('ResourceInfo.descriptionTitle')}>
+          <div className="app-ResourceInfo__description">
+            {resource.description && <WrappedText openLinksInNewTab text={resource.description} />}
```

Comment:
Unnecessary condition here, `resource.description` is guaranteed to be something here since it's already checked for.